pkg/metrics/throttle.go

```diff
@@ -44,34 +79,80 @@ func gatherThrottleMetricsForever(subsystem string) {
 		processCPUThrottledTime,
 	)
 
+	fsys := os.DirFS("/")
```

Comment:
did this instead of just directly using the `os` package functions so I can try to mimic this as closely as possible in tests